docs/projects.md

```diff
@@ -278,3 +281,96 @@ You should see something like this:
+
+### Using Github with a Clone
+A clone is a copy of a repository.  You are free to make changes to the repository and will be able to get an up to date version of the original repository.
+To clone the class repository simply click the green **Code** button in the header of the repository and copy the url presented to you for cloning ```https://github.com/tbretl/ae353-sp21.git```. Make sure you have Git installed on your computer (if not view instructions below under Install Git client tools) and then git bash into the directory where you want to store the local copy of the repository.  Then run the following commands.
+```
+git clone https://github.com/tbretl/ae353-sp21.git
+``` 
+The class repository will then be copied into the directory.
+
+To update the class repository you have a two options: ```fetch``` and ```pull```.
+
+* Fetching will retrieve all the new work in the class repository, but will not merge those changes into your local repository.  This means that your work will not be overwritten.  To fetch simply git bash into the repo directory and run the command ```git fetch origin```.
+* After fetching you can run ```git merge main```, where main is the name of the branch.
+* As a shortcut you can run the ```git pull origin``` command which will complete both fetch and merge.
+
+### Using Github with a Fork
+A fork of the repository is a copy of the repository.  You are free to experiment with your local copy without fear of affecting the original repository.  A fork is useful because unlike a clone, there is still a link between the original repository and your local copy.  This means that you can contribute to the class code by using Pull Requests.
+
+To fork the class repository simply click the **Fork** button in the header of the repository.  This will allow you to create a forked repository on your own Github account.
+Now perhaps you want to be able to access this repository from your computer, no worries.  Make sure you have Git installed on your computer (if not view instructions below under Install Git client tools) and then git bash into the directory where you want to store the local copy of the repository.  Then run the following commands.
+```
+git clone https://github.com/<GITHUB USERNAME>/<REPOSITORY NAME>.git
+
+git remote add upstream https://github.com/tbretl/ae353-sp21.git
+
+git fetch upstream
+```
+
+* Be sure to replace <GITHUB USERNAME> with your Github Username and <REPOSITORY NAME> with the name of the repository, probably ae353-sp21.
```

Comment:
"GITHUB USERNAME" and "REPOSITORY NAME" don't show up outside of the code. I believe those type of brackets are used for something else in GitHub. Try changing to quotes and maybe specify one should also remove the brackets? Also, rather than saying "probably ae353-sp21", say "in this case, ae353-sp21".